VeriD/Issues/vergoedingenpas/description.xml

```diff
@@ -21,7 +21,8 @@
 	<ShouldBeSingleton>false</ShouldBeSingleton>
 	<Attributes>
 		<Attribute id="logo"
-			displayHint="portraitPhoto">
+			displayHint="portraitPhoto"
+			optional="true">
```

Comment:
Please note that `optional` attributes can't go back to non optional attributes.

Reply:
Yes understood..